include/http/request_handler.js

```diff
@@ -164,9 +164,9 @@ module.exports = function RequestHandlerModule(pb) {
           uid: site.uid,
           displayName: site.displayName,
           hostname: site.hostname,
-          defaultLocale: site.defaultLocale,
-          supportedLocales: site.supportedLocales,
-          prevHostnames: site.prevHostnames
+          defaultLocale: site.defaultLocale || pb.Localization.getDefaultLocale(),
+          supportedLocales: site.supportedLocales || [],
```

Comment:
It might be cleaner to ensure defaults in the site service and do error checking here.  thoughts?